src/js/mods/modloader.js

```diff
@@ -104,38 +103,33 @@ export class ModLoader {
     }
 
     exposeExports() {
-        if (G_IS_DEV) {
-            let exports = {};
-            const modules = import.meta.webpackContext("../", {
-                recursive: true,
-                regExp: /\.[jt]s$/,
-                exclude: /\.d\.ts$/,
-            });
-            Array.from(modules.keys()).forEach(key => {
-                /** @type {object} */
-                const module = modules(key);
-                for (const member in module) {
-                    if (member === "default" || member === "__$S__") {
-                        // Setter
-                        continue;
-                    }
-                    if (exports[member]) {
-                        throw new Error("Duplicate export of " + member);
-                    }
+        const exports = {};
+        const modules = import.meta.webpackContext("../", {
+            recursive: true,
+            regExp: /\.[jt]sx?$/,
+            exclude: /\.d\.ts$/,
+        });
 
-                    Object.defineProperty(exports, member, {
-                        get() {
-                            return module[member];
-                        },
-                        set(v) {
-                            module.__$S__(member, v);
```

Comment:
This set method never worked in the first place, right?

Reply:
Correct, it *was* working in development builds, but not production (due to optimization apparently). The ES specifications says that imports are "live references" but they're immutable. Note that `shapez.` exports are *not* live references, which is why some bundled mod code may not work the same way as `shapez.`-prefixed code does.